onnx/backend/test/cpp/driver_test.cc

```diff
@@ -0,0 +1,249 @@
+#include "gtest_utils.h"
+
+#include "onnx/checker.h"
+#include "onnx/onnxifi.h"
+#include "onnx/onnxifi_loader.h"
+#include "onnx/string_utils.h"
+
+// Now we set this macro to be enbaled,
+// but once we have valid onnxifi backend,
+// we should remove it and set it to be a compiler option.
+#define ONNXIFI_DUMMY_BACKEND
```

Comment:
someone could start benefit from this driver_test if they have an implementation of the ONNXIFI interface.

If you allow it via the following:
```
#ifndef USE_ONNXIFI_BACKEND
#define USE_ONNXIFI_BACKEND false
#endif
```

If someone wants to enable ONNXIFI_BACKEND they would compile with -DONNXIFI_BACKEND=true.

Reply:
Yes I guess that is better